src/tools/ping/PingConnection.cpp

```diff
@@ -45,6 +45,20 @@ PingConnection::PingConnection(
     MsQuic->SetCallbackHandler(Connection, (void*)QuicCallbackHandler, this);
 }
 
+PingConnection::PingConnection(
+    _In_ PingTracker* Tracker,
+    _In_ HQUIC Connection
```

Comment:
Why didn't you just modify the old constructor? Also, did you update the existing client callers to pass the non-default param now?